### DISCUSSION TITLE
PIMCORE Bundle Generator Issue

### DISCUSSION THREAD

**Discussion by KasthuriAnandhan** [pimcore/pimcore#17437] [Q&A / Support] [answered]
Hi Team, @kingjia90, @BigJabber, @AlephCode

I have installed a sample bundle through the Pimcore bundle generator on my local Windows machine. After completing all the installation and configuration steps, I tried to access the route specified in DefaultController.php. Instead of getting a response, I'm encountering the following error. Can anyone help me resolve this issue? 

For your reference, I have provided the GitHub repository link as well.

**Github Repository Link :** https://github.com/KasthuriAnandhan/TestCode/
**Route :**  http://localhost/demo_test
**Error :** 
![image](https://github.com/user-attachments/assets/d14abb8c-8d94-4e67-8adf-0df425fa0eb9)

Thanks In Advance,
Kasthuri

**Comment by kingjia90** [comment, marked as answer]
Update: fixed by https://github.com/pimcore/bundle-generator/pull/22 in `v2.0.3`


https://github.com/KasthuriAnandhan/TestCode/blob/main/Demo/TestBundle/config/pimcore/routing.yaml

"DemoTestBundle" should be "Demo/TestBundle"

**Comment by KasthuriAnandhan** [comment]
Thank you so much💐 @kingjia90. I have tried so far to get an output. But I didn't change anything from the installed files; they were generated as is when I installed the bundle. To get this output "**Hello world from demo_test**", Is we need to change in the configuration files?

**Reply by kingjia90** [reply]
I didn't get it, did you mean that my suggestion above worked but you were expecting it would work out of the box as you didn't apply any manual custom changes, right? If yes, i guess i'll have to fix the generator but in the meanwhile wish you a nice weekend

**Comment by KasthuriAnandhan** [comment]
Yes @kingjia90 , your suggestion worked for me. However, I thought that after installing the bundle, we wouldn't need to change anything in the configuration. After making the changes you suggested, I was able to get the response. So, do we need to make any configuration changes after installing the bundle to get the output?

**Reply by kingjia90** [reply]
Thank you for it, i've converted this into an issue and provided a PR that should be fixing it for future generated bundles, but for the current one a manual fix is required
https://github.com/pimcore/bundle-generator/pull/22

**Comment by KasthuriAnandhan** [comment]
Okay, Thank you @kingjia90
